README.md

```diff
@@ -48,6 +48,26 @@ Input:
 }
 ```
 
+#### Index claim
+
+Claims that a content graph can be found in blob(s) that are identified and indexed in the given index CID.
+
+Capability: `assert/index`
+
+Input:
+
+```js
+{
+  content: CID /* CAR CID */,
+  /**
+   * Link to a Content Archive that contains the index.
+   * e.g. `index/sharded/dag@0.1`:
+   * @see https://github.com/w3s-project/specs/blob/main/w3-index.md
+   */
+  index: CID
+}
+```
```

Comment:
One thing that came into my mind while reviewing https://github.com/w3s-project/RFC/pull/29/files is that perhaps requirement that index be contained in a separate CAR is flawed. It might be a good idea to not have that requirement and instead assume that some CAR may contain the index identified by CID, which may not be the root of that CAR (e.g. we could add index in the last content shard instead). In the future where we could potentially represent whole upload with a single giant blob we may have even more reasons to suffix index to the end of it as opposed to having it stored separately.

I'm not sure what exactly implications are here, but  perhaps describing index as pair of container (blob/car) and index root would be more prudent. Also worth calling out this is something we ended up doing in the invocations and receipts.